streams/src/main/java/org/apache/kafka/streams/kstream/internals/KTableTransformValues.java

```diff
@@ -118,11 +120,14 @@ public void process(final Record<K, Change<V>> record) {
 
             if (queryableName == null) {
```

Comment:
Side cleanup: all other processors use `queryableName != null` -- not sure why it's reversed here. Kinda annoying.

Reply:
Updated in https://github.com/apache/kafka/pull/13615.